hippod/hippod_login.py

```diff
@@ -0,0 +1,173 @@
+
+import os
+
+import datetime
+
+import json
+
+from aiohttp import web
+
+
+# Customize the cookie
+# this is more convient way to modify cookie.
+COOKIE_NAME = "OldTamil"
+COOKIE_VALUE = 0
+
+
+# Define host and port.
+HOST = '127.0.0.1'
+PORT = 8080
+
+# Add required files.
+LOGIN_HTML = 'login_system/login.html'
+SITE_HTML = 'login_system/site.html'
+REDIRECT_HTML = 'login_system/redirect.html'
+CONFIG_FILE = 'login_system/configuration.json'
+
+
+# We use ERROR for server_error web response.
+ERROR = '''<!DOCTYPE html>
+<html>
+  <head>
+    <title>SERVER ERROR</title>
+    <style type ="text/css">
+    body { text-align:center; padding: 10%;
+    font-weight: bold; font: 20px Helvetica, sans-serif; }
+    a {padding: 30px;}
+    h1 {color:#FF0000 ; font-family: Arial, Helvetica, sans-serif;
+        font-size: 30px;}
+    </style>
+  </head>
+     <body>
+     <h1> Sorry! Something went wrong </h1>
+     <h2> Internal server error</h2>
+       <p> Please visit us later</p>
+       <p>The Team</p>
+     </body>
+</html>'''
+
+
+class Login:
+    """ Login performs the main task of this system.
+
+    The class runs login_required function which validate the user and gives
+    permission and _check_the_file, _check_credentials, and _load_html_file
+    are responsible for file handling and credentials loading.
+
+    Cookie handling has been implemented.
+    The cookie will be stored on the client site for 30 days long.
+
+
+    """
+
+    def _check_the_file(self, filename):
```

Comment:
Public methods first, the the private ones. This is the flow, think new readers of the source code: you want to start to see the big pictures, then you want to go into the details. Right?